lessons/linuxadmin/installation/info.yml

```
@@ -7,3 +7,5 @@ subpages:
         subtitle: VirtualBox
     gnome-boxes:
         subtitle: Gnome Boxes
+    apple-silicon:
+        subtitle: Instalace na Apple Silicon
```

```suggestion
        subtitle: Instalace na Apple Silicon
```
Chybí znak nového řádku na konci souboru. To není extrémně zle, ale většinou by se editor měl nastavit tak, aby ten znak konce řádku na konec souboru zapisoval.